packages/react/src/I18nProvider.tsx

```diff
@@ -11,13 +12,14 @@ export interface I18nProviderProps extends I18nContext {}
 
 const LinguiContext = React.createContext<I18nContext>(null)
 
+const config = getConfig({ configPath: process.env.LINGUI_CONFIG })
+const [i18nImportModule, i18nImportName = "i18n"] = config.runtimeConfigModule
+
 export function useLingui(): I18nContext {
   const context = React.useContext(LinguiContext)
 
-  if (process.env.NODE_ENV !== "production") {
-    if (context == null) {
-      throw new Error("useLingui hook was used without I18nProvider.")
-    }
+  if (context == null) {
+    return require(i18nImportModule)[i18nImportName]
```

Comment:
I wouldn't bother importing `i18nImportModule` here... This should be only for the simplest cases when you use the default `i18n` from `@lingui/core`. As soon as you start using custom `i18n` instance, you would have to use `I18nProvider`